tdp/cli/utils.py

```diff
@@ -79,6 +80,7 @@ def collections(func: FC) -> FC:
         required=True,
         callback=_collections_from_paths,
         help=f"List of paths separated by your os' path separator ({os.pathsep})",
+        is_eager=True,
```

Comment:
Why adding `eager` ?

Reply:
So that this option is loaded first. It can then be consumed by others, as this one for example:

```py
def _check_component(
    ctx: click.Context, param: click.Parameter, value: Optional[str]
) -> Optional[str]:
    """Click callback that check if the component exists."""
    collections: Collections = ctx.params["collections"]
    service: str = ctx.params["service"]
    if value and value not in [
        sc_name.component_name
        for sc_name in collections.get_components_from_service(service)
    ]:
        raise click.UsageError(
            f"Component '{value}' does not exists in service '{service}'."
        )
    return value
```

I only added `is_eager=True` to the ones I needed. But, I think that we should add it on any "primitive" option that we consider (basically, the one we put in the .env file in my opinion).

Reply:
Add a comment to know why `eager` is added.

Reply:
Done!